java-manta-client-unshaded/src/main/java/com/joyent/manta/exception/MantaErrorCode.java

```diff
@@ -180,4 +532,52 @@ public static MantaErrorCode valueOfCode(final Object object) {
 
         return valueOfCode(object.toString());
     }
+
+    /**
+     * Populates a Inclusive ArrayList for {@link MantaErrorCode} by its code value.
+     *
+     * @return a thread-safe Manta Errors List.
+     * @throws MantaClientException thrown if deprecated fields are not present or due to security violation.
+     */
+    public static List<MantaErrorCode> getNonDeprecatedList() throws MantaClientException {
```

Comment:
Is this still needed? Also why is it a `List` and not a `Set`?

Reply:
I thought a ```List``` would be preferable since it allows duplicated elements while ```Set``` usually contains an unordered collection of distinct elements. Also traversing a Set can only be done in a forward direction whereas ```Lists``` are flexible(```both forward and backward```) in that regard.

Reply:
Don't we want all values to be unique? Ordering for `Set` is an optional concern. There are several implementations of `java.util.NavigableSet` that allow for ordered sets that provide means for bidirectional traversal.

Reply:
```List``` Removed.